wouso/interface/forum/tests.py

```diff
@@ -0,0 +1,22 @@
+from django.test.client import Client
+from wouso.core.tests import WousoTest
+from django.core.urlresolvers import reverse
+from wouso.interface.forum.models import Category
```

Comment:
The import order should be:
- standard library imports
- Django imports
- local application imports

Each category should be separated by a blank line.